### PR TITLE
Display approved feedback on carousel

### PR DESCRIPTION
**Pull request by transient-jonas**

### CODE REVIEW

File: client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          <Row>
+            <ThumbWrapper>
+              {item.answer ? (
+                <ThumbsUpWithoutPadding />
+              ) : (
+                <ThumbsDownWithoutPadding />
+              )}
+            </ThumbWrapper>
+            <Spacer8 />
+            <DateTag>{dayjs(item.createdAt).format('d MMM')}</DateTag>
```
Comment (gewfy) [inline client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: I don't think this is locale safe. Check these :) https://day.js.org/docs/en/display/format#localized-formats

File: content/src/ui/Overlay.Session.json [hunk 2]
```diff
@@ -14,8 +15,9 @@
     "journeyButton": "Min plan",
     "buttonsHeading": "Lägg till i Min plan, kalender eller dela sessionen",
     "deleteButton": "Ta bort session",
-    "coCreatorsHeading": "Samskapat med",
-    "howItWorks": "Så funkar det"
+    "coCreatorsHeading": "Skapad med",
+    "howItWorks": "Så funkar det",
+    "feedbackHeading": "Reviews"
```
Comment (gewfy) [inline content/src/ui/Overlay.Session.json:20]: "Recensioner"

File: content/src/ui/Modal.CreateSession.json [hunk 3]
```diff
@@ -88,6 +89,8 @@
     },
     "noUpcomingSessions": "🌱 Inga planerade sessioner.",
     "description": "Gör sessionen med...",
+    "coCreatorsHeading": "Skapad med",
+    "feedbackHeading": "Reviews",
```
Comment (gewfy) [inline content/src/ui/Modal.CreateSession.json:93]: "Recensioner"

File: client/src/lib/session/hooks/useExerciseFeedbackCount.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      setCount(null);
+    }
+  }, [exerciseId, mode]);
+
+  useEffect(() => {
+    fetchFeedbackCountForExercise();
+  }, [exerciseId, fetchFeedbackCountForExercise]);
+
+  return {
+    count,
```
Comment (gewfy) [inline client/src/lib/session/hooks/useExerciseFeedbackCount.ts:29]: ... and rename this to `rating`

File: functions/src/models/slack/index.ts [hunk 5]
```diff
@@ -188,3 +189,30 @@ export const updatePostMessageVisibility = async (
...
+        ...blocksWithoutAction,
+        createFeedbackVisibilityActionBlock(feedbackId, visible),
+      ],
+      channel: channelId,
+      ts,
+    });
+  } catch (error) {
+    throw new SlackError(SlackErrorCode.couldNotUpdateMessage, error);
+  }
+};
```
Comment (gewfy) [inline functions/src/models/slack/index.ts:218]: 👍

**Review by gewfy — APPROVED** [review]
Looking good! Exciting improvement! :) Approved but needs some adjustments as per comments 🙏

File: client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx (hunk 1 above)
Comment (transient-jonas) [reply client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: Seems like none of those match the design format 😕

Comment (transient-jonas) [reply client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: Shall we update all of these to use locale formats?
Screenshot 2023-09-07 at 15 55 19

Comment (gewfy) [reply client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: Never mind I got it wrong... They do get localised but does not change the order of things based on locale. So this `format('ddd, D MMM HH:mm')` actually gets localised correctly in this screenshot ("tor")

Comment (gewfy) [reply client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: Never mind I got it wrong... They do get localised but does not change the order of things based on locale. So this `format('ddd, D MMM HH:mm')` actually gets localised correctly in this screenshot ("tor")
image

Comment (transient-jonas) [reply client/src/lib/components/FeedbackCarousel/FeedbackCrousel.tsx:75]: aha, cool!

**Pull request merged**